lib/src/flow.cpp

```diff
@@ -17,15 +17,15 @@ using namespace mxl::lib;
 
 extern "C"
 MXL_EXPORT
-mxlStatus mxlCreateFlow(mxlInstance instance, char const* flowDef, char const* /*options*/, mxlFlowInfo* flowInfo)
+mxlStatus mxlCreateFlow(mxlInstance instance, char const* flowDef, char const* options, mxlFlowInfo* flowInfo)
 {
     try
     {
         if (flowDef != nullptr)
         {
             if (auto const cppInstance = to_Instance(instance); cppInstance != nullptr)
             {
-                auto const flowData = cppInstance->createFlow(flowDef);
+                auto const flowData = cppInstance->createFlow(flowDef, options ? options : std::string());
```

Comment:
Feels a bit off doe to the implicit conversion of the true case, maybe `std::string{options ? options : ""}` instead?!